src/models/experts/reducers.ts

```diff
@@ -1,19 +1,33 @@
 /* eslint-disable */
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 import { LoadingStatusEnum } from '../../old/lib/types';
-import { IExpertsState } from './types';
-import { fetchExperts } from './asyncActions';
+import {
+  IExpertsState,
+  AutorsListOrder,
+  AutorsListSortBy,
+  ISortAutorsList,
+  ITextFields,
+} from './types';
+import { fetchExperts, fetchExpertsAutorsList } from './asyncActions';
 import { getAsyncActionsReducer } from '../helpers/asyncActions';
 
 const initialState: IExpertsState = {
   data: {
     expertIds: [],
     experts: {},
-    meta: {
-      totalPages: 0,
-      pageNumber: 0,
-      isLastPage: false,
-      totalElements: 0,
+    totalPages: 0,
+    isLastPage: true,
+    totalElements: 0,
```

Comment:
why this information has been moved from `meta`? I understand that "total experts number" and "current page" are _different_ metadata, but still it's not like specific expert's name or region(which are definitely `data`)

Reply:
or is this changed due to changes on backend side?